### PR TITLE
Add display of selected autonomous path

### PR DESCRIPTION
**Pull request by KangarooKoala**
Hopefully this will help with learning the system behind the auto names.

I included methods to pre-load everything in case there were performance concerns, but they probably won't be necessary (especially since we shouldn't be changing the selected auto during matches).

### CODE REVIEW

**Review by jbko6 — DISMISSED** [review]

File: src/main/java/frc/team2412/robot/util/PathPlannerAutos.java
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/util/PathPlannerAutos.java]: Consider using one of the Json libraries wpilib uses internally for parsing Json instead of writing your own simple parser: https://github.com/FasterXML/jackson or https://github.com/google/gson
Not blocking for this PR, but would help make this more resilient and ideally re-use code already linked into the program.

**Review by jamesdooley4 — DISMISSED** [review]

File: src/main/java/frc/team2412/robot/util/PathPlannerAutos.java
Comment (KangarooKoala) [reply src/main/java/frc/team2412/robot/util/PathPlannerAutos.java]: Oops, I was worried about adding a dependency but forgot WPILib already uses some JSON libraries. I'll switch to using FasterXML/jackson tomorrow.

**Comment by KangarooKoala** [comment]
It looks like loading might be a little bit slower (as in not instant), but it should be fine- It was just a little stutter that I'm not even 100% sure was caused by this change.

**Comment by KangarooKoala** [comment]
Regarding the TODO for the named commands, there's work in progress on conditional paths that would be implemented through those named commands, but I think I'll add the logic for that after that gets merged in. It isn't high importance, so I don't think this PR should be blocked on that.

**Review by TAKBS2412 — APPROVED** [review]

**Review by jamesdooley4 — APPROVED** [review]

**Pull request merged**
